src/coreclr/src/tools/crossgen2/crossgen2/Program.cs

```diff
@@ -36,17 +36,6 @@ private Program(CommandLineOptions commandLineOptions)
             _commandLineOptions = commandLineOptions;
         }
 
-        private void Help(string helpText)
-        {
-            Console.WriteLine();
-            Console.Write("Microsoft (R) CoreCLR Native Image Generator");
```

Comment:
I assume this became dead code after some refactoring. Do we need to track adding the logo back somewhere? All Microsoft command line tools I know of print what they are, so maybe this is mandated somewhere.

Reply:
`Microsoft (R) ...` used to be mandatory for closed source command line tools. We have removed printing of `Microsoft (R) ...` for open source command line tools.

Reply:
This code has been dead since we switched to the new command line library, which produces its own help spew.